### DISCUSSION TITLE
Improving precision

### DISCUSSION THREAD

**Discussion by lirilkumar** [ultralytics/yolov5#6200] [Q&A] [answered]
i want to improve precision, with little less recall.
i am looking for setting where we cans setup confidence level which will be used to classify whether the identified box is matching with any object or not.
Can someone help me how can we increase that threshold?
And what it will change in training and in production?
@glenn-jocher

**Comment by glenn-jocher** [comment, marked as answer]
@lirilkumar confidence threshold is an inference parameter, i.e.:

```bash
python detect.py --weights yolov5s.pt --conf 0.25
```

**Reply by lirilkumar** [reply]
@glenn-jocher  aah!
any way to get high precision over recall in training?

**Reply by glenn-jocher** [reply]
@lirilkumar no. You decide your own precision at inference time via the --conf threshold.
